src/hooks/subscription-hooks.js

```diff
@@ -361,3 +361,26 @@ export function useTotalRewardsSubscription() {
 
   return { data: rewards, error }
 }
+
+/**
+ * Queries for the last withdrawal fee movement time made by the given juror
+ * @param {String} jurorId Address of the juror
+ * @returns {Object} Juror's last withdrawal fee movement
+ */
+export function useJurorLastWithdrawalTimeSubscription(jurorId) {
+  const [{ data }] = useSubscription({
+    query: JurorLastFeeWithdrawal,
+    variables: { owner: jurorId?.toLowerCase() },
+    pause: !jurorId,
```

Comment:
If you switch accounts it would resubscribe again as usual.

> I was imagining you would just not subscribe if the jurorId wasn't given.

Yeah that's the intention of the `pause` attribute!